example/declarative.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+library grinder.example.declarative;
+
+import 'package:grinder/grinder.dart';
+
+const main = grind;
```

Comment:
This code is correct, but I think the example will read simpler if it's 
```
main() => grind();
```

And an individual user can do the funky assignment in their scripts if they choose.